.github/workflows/conformance-externalworkloads.yaml

```diff
@@ -68,7 +68,7 @@ env:
   vmName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-vm
   zone: us-west2-a
   vmStartupScript: .github/gcp-vm-startup.sh
-  cilium_cli_version: v0.12.4
+  cilium_cli_version: v0.12.7
```

Comment:
Why didn't you update other workflows?

Reply:
Just to check whether the previous failure was fixed. https://github.com/cilium/cilium/pull/22140 is supposed to update all workflows.